tests/transports-governance/test_helpers.py

```
@@ -0,0 +1,644 @@
+        if include_requests:
+            config.update(
+                {
+                    "request_max_limit": random.randint(10, 10000),
+                    "request_reset_duration": random.choice(["1m", "5m", "1h", "1d"]),
+                }
+            )
+
+        return config
+
+    @staticmethod
+    def generate_customer_data(include_budget: bool = False) -> Dict[str, Any]:
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)